src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier(new ColorSupplier());
+        Figure[] figures = new Figure[6];
+
+        // Generate random figures
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        // Draw each figure
+        for (Figure figure : figures) {
+            figure.draw();
+        }
```

Comment:
- one for loop is enough
- you should make half of the figures default (white circles of radius 10), and half - random